share/change.py

```diff
@@ -108,12 +130,15 @@ def __init__(self, node, disambiguate=True, extra_namespace=None):
         if disambiguate:
             self._disambiguate()
 
-    def resolve_relations(self, mapper):
+    def resolve_through_relations(self, mapper):
         for key, relations in self.__reverse_relations.items():
-            resolved = [
-                next(x for x in mapper[r['@id'], r['@type'].lower()].related if x['@id'] != self.id)
-                for r in relations
-            ]
+            resolved = []
```

Comment:
Oh, that's an artifact from when I made identifiers many-to-one; sometimes `next` would be given an empty sequence and raise `StopIteration`. The change should be equivalent except for handling that case gracefully.